index.qmd

```diff
@@ -341,4 +351,94 @@ Founded last year:
 
 ## Practical daily development process
 
-Doug would you like to add here a few slides?
+Let's take a look at what it looks like in action. 
+
+> **Case Study:** `simulate()` feature
+> We'll follow the addition of a major `v0.3.0` feature, tracking its progress 
+> our team's workflow.
+
+## 1. Open Communication During Design
+
+::: columns
+::: {.column width="30%"}
+Features start with a discussion to make sure we're aligned with the need and 
+approach.
+:::
+
+::: {.column width="70%"}
+![](resources/dev-feature-request.png)
+:::
+:::
+
+## 2. Live Discussion at Bi-weekly Call
```

Comment:
it is actually weekly or even twice weekly in the past, but we might adjust the frequency now down

Reply:
Ah okay - I saw biweekly in a previous slide. I'll merge for now just to keep things moving quickly, but we can definitely reword this.